code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -455,7 +455,7 @@
 	chosen_mortal.update_mobility()
 
 	return .
```

Comment:
```suggestion
	return
```
bruh what

Reply:
not related to this PR, plus you could just remove it entirely.